src/components/Sensor/sensor.css

```diff
@@ -0,0 +1,50 @@
+@font-face {
+  font-family: 'OpenSans-Regular';
+  src: local('OpenSans-Regular'), url(../../fonts/OpenSans-Regular.ttf);
+}
+
+.sensor-font {
+  font-family: 'OpenSans-Regular';
```

Comment:
should we choose a fallback font since that is not a standard font and it may take a while to load and idk, maybe it fails somehow?

Reply:
The fallback font by default will be sans-serif